modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryRelatedObjectsResourceTest.java

```diff
@@ -1234,14 +1236,22 @@ private void _assertPagination(
 
 		JSONObject jsonObject = HTTPTestUtil.invokeToJSONObject(
 			null,
-			_getEndpoint(objectRelationship.getName()) + "?page=1&pageSize=1",
+			_getEndpoint(objectRelationship.getName()) + "?page=1&pageSize=2",
 			Http.Method.GET);
 
-		_assertEquals(baseModel, jsonObject.getJSONArray("items"));
+		Set<Long> ids = new HashSet<>();
 
-		Assert.assertEquals(2, jsonObject.getLong("lastPage"));
+		for (Object item : jsonObject.getJSONArray("items")) {
+			JSONObject itemJSONObject = (JSONObject)item;
+
+			ids.add(itemJSONObject.getLong("id"));
+		}
+
+		Assert.assertTrue(ids.contains(baseModel.getPrimaryKeyObj()));
+
+		Assert.assertEquals(1, jsonObject.getLong("lastPage"));
 		Assert.assertEquals(1, jsonObject.getLong("page"));
-		Assert.assertEquals(1, jsonObject.getLong("pageSize"));
+		Assert.assertEquals(2, jsonObject.getLong("pageSize"));
 		Assert.assertEquals(2, jsonObject.getLong("totalCount"));
 
 		jsonObject = HTTPTestUtil.invokeToJSONObject(
```

Comment:
I'm uncomfortable with this solution because of this Assert.
I don't think that we are testing pagination since our results with and without pageSize are the same.
What is your opinion?

Reply:
(eu não consegui marcar o comentário na linha correta, falo na solução na totalidade)

Reply:
That makes sense. I changed things up a bit to make sure we are testing pagination.